docs/devel-quick-start.md

```diff
@@ -11,7 +11,7 @@ SPDX-License-Identifier: Apache-2.0
 1. Create a branch from where you want to base your work (usually main).
 1. Make your changes and arrange them in readable commits.
 1. Make sure your commit messages are in the proper format.
-1. Make sure all [tests](docs/testing.md) pass, and add any new [tests](docs/testing.md)
+1. Make sure all [tests](./testing.md) pass, and add any new [tests](./testing.md)
```

Comment:
The current link is not broken, try here:
https://github.com/RamenDR/ramen/blob/main/docs/devel-quick-start.md

Maybe you want to improve the links to use shorter form? Please improve the commit message to make your intent more clear.

Reply:
Rest of the links are ok but this one is broken. My mistake when I refactored the devel-quick-start.md file.

Reply:
@nirs Thanks for reviewing. While accessing both the links I am getting this result.

<img width="1392" alt="Screenshot 2024-03-19 at 8 47 28 AM" src="https://github.com/RamenDR/ramen/assets/17064454/2bba2084-df1b-424b-9264-751607379243">